### ISSUE TITLE
Cant pull images

### ISSUE THREAD

**Issue by karloscodes** [docker/docker-registry#325]
I have setup samalba/docker-registry from github source code following official instructions, everything seems to be working fine on the installation, but when it comes to pull previously pushed images I'm getting some errors, let me explain myself:

I have ubuntu:12.04 image on my development machine and I have created a Dockerfile with the intention of creating a base image for my company based on ubuntu:12.04, to build this image I run this command: (docker-index.my.com is my private repository)

`docker build -t docker-index.my.com/base .`
or this one:

`docker build -t docker-index.my.com/base:1.0 .`
to push the images I use:

`docker push docker-index.my.com/base:1.0`
with or without the version tag (:1.0)

On other PC when I try to pull somehow those images from the private repository I get the following:

`docker run -i -t docker-index.my.com/base /bin/bash`
which outputs:

Unable to find image 'docker-index.my.com/base' locally
Pulling repository docker-index.my.com/base
2014/04/22 09:00:38 Error: No such image: docker-index.my.com/base (tag: latest)
Basically this is my workflow. I have checked and docker-registry seems to have the image with the tag I used to push it, and also it seems to be receiving the request from the docker clients but I can get the pulls working whether I use tag or not. Am I missing something here? what can it do to find/fix the problem?
